packages/ember-toucan-form/src/-private/input-field.ts

```diff
@@ -0,0 +1,59 @@
+import Component from '@glimmer/component';
+import { assert } from '@ember/debug';
+import { action } from '@ember/object';
+
+import type { HeadlessFormBlock, UserData } from './types';
+import type { ToucanFormInputFieldComponentSignature as BaseInputFieldSignature } from '@crowdstrike/ember-toucan-core/components/form/fields/input';
+import type { FormData, FormKey, ValidationError } from 'ember-headless-form';
+
+type ComponentArguments<
```

Comment:
What's reason for separating the args (giving it an explicit separate type) from the overall signature? Nothing wrong about that, just wondering as you need quite a bit of boilerplate (e.g. the duplicated generics in L10-11).

Reply:
None at all! Went ahead and collapsed everything.  Thanks for the suggestion!